assets/css/style.css

```diff
@@ -0,0 +1,378 @@
+html {
+    /* background-color: black; */
+    scroll-behavior: smooth;
+
+}
+
+body{
+    overflow-x: hidden;
+}
+
+#page-container { 
+    width:100%;
+    background-image: url("../images/retrowave-background.png");
+    background-position: top;
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    border: solid thick red;
+    box-shadow: 0 0 100px black;
+}
+
+#profile-section h1{
+    width:100%;
+}
+
+.portfolio-section {
+    border: solid 5px red;
+    box-shadow: 0 0 10px red;
+    margin-top: 20px;
+    position: relative;
+}
+
+#toggle-button {
+    position: absolute;
+    left: 40px;
+}
+
+.section-title {
+    position: absolute;
+    width: 100%;
+    top: -20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.section-title h2 {
+    color: white;
+    width: fit-content;
+    text-align: center;
+    padding: 10px;
+    border: solid 1px red;
+    background-color: black;
+    box-shadow: 0 0 10px red;
+}
+
+#profile-section {
+    margin-top: 10px;
+    padding-top: 50px;
+    text-align: center;
+    background-position: top;
+}
+
+#profile-section h1 {
+    font-family: Droid;
+    margin-top: 100px;
+    text-align: middle;
+    color: white;
+}
+
+#profile-section nav ul {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    list-style-type: none;
+}
+
+#profile-section nav li {
+    margin: 10px;
+    /* margin: 10px; */
+}
+
+#profile-section nav li a {
+    font-family: 'Tomorrow', sans-serif !important;
+    color: white;
+}
+
+#profile-section nav li a:hover {
```

Comment:
great use of on hover effects - this really contributes to creating a sleek UI :-)